publisher-sdk/src/main/java/com/criteo/publisher/adview/MraidInteractor.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher.adview
+
+import android.webkit.WebView
+import com.criteo.publisher.annotation.OpenForTesting
+import com.criteo.publisher.logging.LoggerFactory
+
+@OpenForTesting
+internal class MraidInteractor(private val webView: WebView) {
+
+  private val logger = LoggerFactory.getLogger(MraidInteractor::class.java)
+
+  fun notifyReady() {
+    callOnMraidObject("notifyReady()")
+  }
+
+  fun notifyError(message: String, action: String? = null) {
+    callOnMraidObject("notifyError(${asJsArgs(message, action)})")
```

Comment:
nit: you could use suffix functions on string to get something like:

```
"notifyError".jsCall(message, action)
```

Then `jsCall` would to the same as your `asJsArgs` but add also the parenthesis.
Or it could even call directly the `callOnMraidObject` ?

Reply:
What do you think about making it invoke operator. It will look like this 
```
"notifyError"(message, action)
```

Reply:
LGTM